packages/ndla-ui/src/locale/messages-nn.ts

```diff
@@ -492,6 +493,11 @@ const messages = {
       description: 'Hugs å kopiera teksten som skal leggjast ved lydfila der du bruker ho.',
       rules: 'Reglar for bruk av lydfila:',
     },
+    podcast: {
+      heading: 'Slik gjenbruker du podkaster',
```

Comment:
gjenbruk/gjenbrukar eller ombruk/ombrukar

Reply:
"Gjenbruker" brukes for alle andre typer. Det ser du om du ekspanderer i begge retninger.

Reply:
Om du vil finne ut om du skal ha 'ar' eller 'er' i et verb, så ser du på fortidsforma av verbet. "I går gjenbrukte eg et bord". Her er endinga på verbet 'e' så derfor er det korrekte 'gjenbruker' eller 'ombruker'. Dagens nynorsk-kurs.

Reply:
Jeg så på bøying av substantiv for liknende ord, det var jo litt feil 😅